source/_integrations/modbus.markdown

```diff
@@ -686,6 +686,16 @@ climates:
         value 1 is written."
```

Comment:
This piece of the description should also updated

Reply:
Done